### PR TITLE
Demonstrating Proof-of-Possession

### PR DESCRIPTION
**Pull request by susanharper-okta**
## Description:
- **What's changed?** Demonstrating Proof-of-Possession guide
- **Is this PR related to a Monolith release?** Yes 2023.03.1

3-10-23
Netlify: https://640bbd6776710f16c21032af--reverent-murdock-829d24.netlify.app/docs/guides/dpop/main/

### Resolves:

* [OKTA-574697](https://oktainc.atlassian.net/browse/OKTA-574697)

### CODE REVIEW

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx Scorecards

Acrolinx checked the content changed here.

Please review the scorecards for each file and make the recommended
improvements.

| Article | Score | Issues | Scorecard | Status |
| ------- | ----- | ------ | --------- | ------ |
| [packages/@okta/vuepress-site/docs/guides/dpop/index.md](https://github.com/okta/okta-developer-docs/blob/7e6872d598f8307fea0a054ec28cb7754ae8eb13/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fdpop%2Findex.md) | 100 | 6 | [→](https://okta.acrolinx.cloud/api/v1/checking/scorecards/11acbfce-5ba2-4b9e-a1c5-8b08d3827564) | [:white_check_mark:](# "Successfully processed") |
| [packages/@okta/vuepress-site/docs/guides/dpop/main/index.md](https://github.com/okta/okta-developer-docs/blob/7e6872d598f8307fea0a054ec28cb7754ae8eb13/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fdpop%2Fmain%2Findex.md) | 82 | 90 | [→](https://okta.acrolinx.cloud/api/v1/checking/scorecards/aabb0601-d864-4775-b8f6-9dd43f9488be) | [:white_check_mark:](# "Successfully processed") |
| [packages/@okta/vuepress-site/docs/guides/index.md](https://github.com/okta/okta-developer-docs/blob/7e6872d598f8307fea0a054ec28cb7754ae8eb13/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Findex.md) | 74 | 29 | [→](https://okta.acrolinx.cloud/api/v1/checking/scorecards/9d33e8af-a79b-4c16-aaab-bce7ef6444ec) | [:white_check_mark:](# "Successfully processed") |

Successfully checked 3 of 3 documents.
See summary in [Content Analysis Dashboard](https://okta.acrolinx.cloud/api/batch/gen.github.f2cbf5cf-f5ad-4f99-be20-d0ff478027bd?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

Reopen the pull request or push new changes to check again.

Depending on the Acrolinx server configuration, the
links expire after some time and you must have a login for the
Acrolinx server to access them again.

File: packages/@okta/vuepress-site/docs/guides/dpop/main/index.md [hunk 1]
```diff
@@ -0,0 +1,330 @@
...
+  "name":"oidc_client",
+  "label":"[Dev App] SPA Client",
+  "status":"ACTIVE",
+  "settings": {
+    "oauthClient": {
+      "response_types":["code"],
+      "grant_types":[
+"authorization_code",
+"refresh_token"
+],
```
Comment (nakuljoshi-okta) [inline packages/@okta/vuepress-site/docs/guides/dpop/main/index.md:98]: fix the alignment

**Review by nakuljoshi-okta — APPROVED** [review]

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx score **DRAFT 06.01.22**
 
A minimum Acrolinx Score of **40** is required. The total score is an average of the subscores.

Select **Total score** to review the Acrolinx scorecard for your article. Try to increase your individual scores, ex. Correctness. Your article will be clearer and more consistent with Microsoft standards.
 
| Article | **Total score**<br />**Required:80** | Word and phrases<br />(Brand, terms)<br />Preferred: 40 | Correctness<br />(Spelling, grammar)<br />Preferred: 40 | Clarity<br />(Readability)<br />Preferred: 10 | Inclusive language<br />(+ accesibility)<br />Preferred: 10 |
|---------|:------------------------------------:|:--------------------------------------------------------:|:--------------------------------------------------------:|:---------------------------------------------:|:------------------------------------------------------------:|
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/dpop/index.md](https://github.com/okta/okta-developer-docs/blob/60d8ce72e144b7001120b4847c01eb832ff47c53/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fdpop%2Findex.md) | [86](https://okta.acrolinx.cloud/api/v1/checking/scorecards/f78cf7de-7d3c-4cc2-85bc-5aeab76cca81) | 100  | 100  | 100  | 100  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/dpop/main/index.md](https://github.com/okta/okta-developer-docs/blob/60d8ce72e144b7001120b4847c01eb832ff47c53/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fdpop%2Fmain%2Findex.md) | [80](https://okta.acrolinx.cloud/api/v1/checking/scorecards/74718473-6522-45bb-88d5-a72bac94b944) | 68  | 87  | 60  | 100  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/index.md](https://github.com/okta/okta-developer-docs/blob/60d8ce72e144b7001120b4847c01eb832ff47c53/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Findex.md) | [75](https://okta.acrolinx.cloud/api/v1/checking/scorecards/b609a7c9-d0f8-4baa-a711-0dfc7d6cb49c) | 48  | 74  | 74  | 85  |

  - The results from this response were procced by the [hugo-targets1.solution.acrolinx.sh](https://hugo-targets1.solution.acrolinx.sh) Acrolinx Cloud Host.
  - Scorecard links by default are only good for 7 days. [Link for this commit's Content Analysis Dashboard](https://okta.acrolinx.cloud/api/batch/gen.github.96a0d4b3-f5ee-4b96-9a73-efc6e2391bff?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

**Review by venkatviswanathan-okta — APPROVED** [review]

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx score **DRAFT 06.01.22**
 
A minimum Acrolinx Score of **40** is required. The total score is an average of the subscores.

Select **Total score** to review the Acrolinx scorecard for your article. Try to increase your individual scores, ex. Correctness. Your article will be clearer and more consistent with Microsoft standards.
 
| Article | **Total score**<br />**Required:80** | Word and phrases<br />(Brand, terms)<br />Preferred: 40 | Correctness<br />(Spelling, grammar)<br />Preferred: 40 | Clarity<br />(Readability)<br />Preferred: 10 | Inclusive language<br />(+ accesibility)<br />Preferred: 10 |
|---------|:------------------------------------:|:--------------------------------------------------------:|:--------------------------------------------------------:|:---------------------------------------------:|:------------------------------------------------------------:|
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/dpop/index.md](https://github.com/okta/okta-developer-docs/blob/79b2b43ec4f9d21df9a13b6d6a582268f773372d/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fdpop%2Findex.md) | [86](https://okta.acrolinx.cloud/api/v1/checking/scorecards/94a68090-2897-4d43-b883-2f7c4517739b) | 100  | 100  | 100  | 100  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/dpop/main/index.md](https://github.com/okta/okta-developer-docs/blob/79b2b43ec4f9d21df9a13b6d6a582268f773372d/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fdpop%2Fmain%2Findex.md) | [80](https://okta.acrolinx.cloud/api/v1/checking/scorecards/281ea21c-6185-424a-8ac8-4c83182b0282) | 68  | 87  | 60  | 100  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/index.md](https://github.com/okta/okta-developer-docs/blob/79b2b43ec4f9d21df9a13b6d6a582268f773372d/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Findex.md) | [75](https://okta.acrolinx.cloud/api/v1/checking/scorecards/8b09cd08-cf74-46f9-aabf-9f1da81f2f1e) | 38  | 74  | 74  | 85  |

  - The results from this response were procced by the [hugo-targets1.solution.acrolinx.sh](https://hugo-targets1.solution.acrolinx.sh) Acrolinx Cloud Host.
  - Scorecard links by default are only good for 7 days. [Link for this commit's Content Analysis Dashboard](https://okta.acrolinx.cloud/api/batch/gen.github.224e9701-9665-4631-8ffb-aa1719dc63ae?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx score **DRAFT 06.01.22**
 
A minimum Acrolinx Score of **40** is required. The total score is an average of the subscores.

Select **Total score** to review the Acrolinx scorecard for your article. Try to increase your individual scores, ex. Correctness. Your article will be clearer and more consistent with Microsoft standards.
 
| Article | **Total score**<br />**Required:80** | Word and phrases<br />(Brand, terms)<br />Preferred: 40 | Correctness<br />(Spelling, grammar)<br />Preferred: 40 | Clarity<br />(Readability)<br />Preferred: 10 | Inclusive language<br />(+ accesibility)<br />Preferred: 10 |
|---------|:------------------------------------:|:--------------------------------------------------------:|:--------------------------------------------------------:|:---------------------------------------------:|:------------------------------------------------------------:|
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/dpop/index.md](https://github.com/okta/okta-developer-docs/blob/ba695cdacab20803a39a459db7519117af8306e7/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fdpop%2Findex.md) | [86](https://okta.acrolinx.cloud/api/v1/checking/scorecards/37c24635-a038-4683-a9b2-3f3a692f618e) | 100  | 100  | 100  | 100  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/dpop/main/index.md](https://github.com/okta/okta-developer-docs/blob/ba695cdacab20803a39a459db7519117af8306e7/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fdpop%2Fmain%2Findex.md) | [82](https://okta.acrolinx.cloud/api/v1/checking/scorecards/33384a9b-7849-4703-8d14-4a832c7476b8) | 87  | 87  | 60  | 100  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/index.md](https://github.com/okta/okta-developer-docs/blob/ba695cdacab20803a39a459db7519117af8306e7/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Findex.md) | [75](https://okta.acrolinx.cloud/api/v1/checking/scorecards/4f36a555-a2c9-4007-9d33-cd5dc037eb7a) | 38  | 74  | 74  | 85  |

  - The results from this response were procced by the [hugo-targets1.solution.acrolinx.sh](https://hugo-targets1.solution.acrolinx.sh) Acrolinx Cloud Host.
  - Scorecard links by default are only good for 7 days. [Link for this commit's Content Analysis Dashboard](https://okta.acrolinx.cloud/api/batch/gen.github.4f04ceeb-f633-4a7a-9346-6f449581bf55?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

**Comment by brianduffield-okta** [comment]
Merging into release branch

**Pull request merged**